hw/ip/entropy_src/README.md

```diff
@@ -28,23 +28,24 @@ A noise source and its relation to an entropy source are defined by [SP 800-90B.
 
 ## Description
 
-This IP block provides an entropy source that is capable of using a PTRNG noise source to generate random values in a manner that is compliant both with FIPS (though NIST SP 800-90B) and CC (AIS31) recommendations.
+This IP block provides an entropy source that is capable of using a PTRNG (Physical True Random Number Generator) noise source to generate random values in a manner that is compliant both with FIPS (though NIST SP 800-90B) and CC (AIS31) recommendations.
 
-<!-- URLS for the following pages not upstream yet, but coming -->
-The random values generated by this block serve as non-deterministic seeds for the CSRNG block.
+At a high-level, the ENTROPY_SRC block, when enabled, will continuously collect entropy bits from the PTRNG noise source, perform health tests on the collected entropy bits, pack them, send them through a conditioning unit, and finally store them into a FIFO.
+The content of this FIFO can either be sent out through a hardware interface or alternatively (but not simultaneously) read by firmware over the TL-UL bus.
+The random values generated by the ENTROPY_SRC block serve as non-deterministic seeds for the CSRNG block.
 The outputs of the CSRNG are then used either directly by firmware or are distributed to other hardware blocks through the Entropy Distribution Network.
 
 In the terms of AIS31 classes, this block is meant to satisfy the requirements for a PTG.2 class physical entropy source, with "internal" entropy (an AIS31 term, meaning the min-entropy as measured just *before the output pins*) exceeding 0.997 entropy-bits/output-bit.
 In NIST terms, this block satisfies the requirements for a "full-entropy" source, which requires even smaller deviations from ideal entropy, at the level of less than one part in 2<sup>64</sup>.
-When operating in full-compliance mode, raw outputs from a PTRNG noise source (external to this block) are passed through a conditioning function based on a suitable secure hash function (SHA-2 or SHA-3) which has been vetted by NIST to meet these stringent requirements.
+When operating in full-compliance mode, raw outputs from a PTRNG noise source (external to this block) are passed through a conditioning function based on a suitable secure hash function (SHA-3) which has been vetted by NIST to meet these stringent requirements.
```

Comment:
Since conditioning function is applied after health checks (according  to line 33),  it might be worth to describe after health checks to avoid confusion.  Or rephrase to "raw outputs from a PTRNG noise source  after health checks"

Reply:
Thanks for the feedback @vsukhoml . Your suggestion makes sense and I've included it.

Reply:
thanks!